### PR TITLE
[translate] finish and update Chinese translate file

### PR DESCRIPTION
**Pull request by zhouyao1994**
[translate] finish and updateC hinese translate file and a add some lables that have been use in show Database View

### CODE REVIEW

**Comment by zhouyao1994** [comment]
[Translate] translate into chinese

**Comment by xrmx** [comment]
Rebase against master please, you are reverting recently updated files.

File: babel/messages.pot [hunk 1]
```diff
@@ -2477,13 +2477,13 @@ msgid "Choose a target"
 msgstr ""
 
 #: superset/assets/javascripts/explore/stores/visTypes.js:828
-msgid "ISO 3166-2 codes of region/province/department"
+msgid "ISO 3166-1 codes of region/province/department"
```
Comment (xrmx) [inline babel/messages.pot]: I don't think you want to change this

File: babel/messages.pot [hunk 2]
```diff
@@ -2477,13 +2477,13 @@ msgid "Choose a target"
...
 msgstr ""
 
 #: superset/assets/javascripts/explore/stores/visTypes.js:829
 msgid ""
-"It's ISO 3166-2 of your region/province/department in your table. (see "
-"documentation for list of ISO 3166-2)"
+"It's ISO 3166-1 of your region/province/department in your table. (see "
```
Comment (xrmx) [inline babel/messages.pot]: and this

**Review by xrmx — CHANGES_REQUESTED** [review]

File: babel/messages.pot (hunk 1 above)
Comment (zhouyao1994) [reply babel/messages.pot]: sorry since i delected this branch ,so i re commit a PR in #3717

**Pull request closed without merging**